web/css/source/_theme.less

```diff
@@ -102,7 +98,43 @@ p,li,dt,dd,td,th,details,button,form,a,div,details {
       margin-left: auto;
     }
   }
+  .panel.wrapper{
+    border-bottom: none;
+  }
+}
+// Luma minor hacks
+.copyright{
+  background-color: @color-blue1;
+}
+
+.breadcrumbs{
+  ul{
+    color:@color-blue1;
+    li{
+      color: @color-blue1;
+      a{
+        color: @color-blue1;
+        :visited{
+          color: @color-blue1;
```

Comment:
Haha! Yep, and believe or not, everyone one of those nested declarations is needed.